cmd/serve.go

```diff
@@ -408,13 +424,17 @@ func (c *ServeCommand) Command(serverService ServerServiceInterface, monitorServ
 			}
 			serveOpts.AnchorPlatformAPIService = apAPIService
 
-			// Starting Scheduler Service (background job)
-			log.Ctx(ctx).Info("Starting Scheduler Service...")
-			schedulerJobRegistrars, err := serverService.GetSchedulerJobRegistrars(ctx, serveOpts, schedulerOptions, apAPIService)
-			if err != nil {
-				log.Ctx(ctx).Fatalf("Error getting scheduler job registrars: %v", err)
+			// Starting Scheduler Service (background job) if enabled
+			if serveOpts.EnableScheduler {
+				log.Ctx(ctx).Info("Starting Scheduler Service...")
```

Comment:
wanna add some additional debugging info?  eg some of the serveOpts props?

Reply:
Individual jobs do their own logging. The only other relevant Opts is `MaxInvitationSMSResendAttempts`, but that gets printed when relevant in `SendReceiverWalletInviteService`